docker_entrypoint.sh

```diff
@@ -105,4 +120,4 @@ fi
 
 trap _term SIGTERM
 
-wait -n $nextcloud_process
\ No newline at end of file
+wait -n $nextcloud_process
```

Comment:
I see nextcloud_process is referenced in lines 7 and 123 but not assigned anywhere. I guess line 80 would be a good candidate but tini is being used.

Reply:
can you propose a solution to be merged?